chia/cmds/coins.py

```diff
@@ -64,20 +62,20 @@ def list_cmd(
     fingerprint: int,
     id: int,
     show_unconfirmed: bool,
-    min_coin_amount: str,
-    max_coin_amount: str,
-    excluded_coin_ids: Tuple[str],
-    excluded_coin_amounts: Tuple[int],
+    min_amount: str,
+    max_amount: str,
+    coins_to_exclude: Tuple[str],
+    amounts_to_exclude: Tuple[int],
```

Comment:
`str`?  In XCH?  And is it really a `Tuple`?  Or a `List`.

Reply:
I believe it should be a List.

Reply:
Changes made Tuple->List. I believe they should all be Lists as changed